### PR TITLE
feat: integrate kapa into apisix.apache.org website

### PR DESCRIPTION
**Pull request by zll600**
We want to try to improve developer experience by integrating kapa into apisix.apache.org. So this pr want to go live right away for a live deployment test.

cc @moonming

### CODE REVIEW

File: blog/en/docusaurus.config.js [hunk 1]
```diff
@@ -105,6 +105,19 @@ module.exports = {
...
+  scripts: [
+    {
+      src: 'https://widget.kapa.ai/kapa-widget.bundle.js',
+      'data-website-id': '23b59d9a-682e-4c3d-9e83-bf2ee85cdc19',
+      'data-project-name': 'APISIX',
+      'data-project-color': '#E8442E',
+      'data-project-logo':
+        'https://user-images.githubusercontent.com/52855735/62005987-53faae80-b16d-11e9-96cc-9dba3470f5b2.png',
+      'data-modal-disclaimer': 'This is a custom LLM for APISIX with access to all developer documentation, GitHub issues and discussions.',
+      'data-modal-example-questions': 'Does APISIX have a UI?,How do I use REGEX for uri routes?',
```
Comment (moonming) [inline blog/en/docusaurus.config.js]: can we set more questions?
- What is Apache APISIX?
- What protocols can Apache APISIX handle?
- How to proxy WebSocket connection?
- How does Apache APISIX work with Keycloak?

Comment (zll600) [reply blog/en/docusaurus.config.js]: OK. I will add more questions.

Comment (zll600) [reply blog/en/docusaurus.config.js]: I think we can follow the user journey.
> What is APISIX -> How to deploy APISIX -> How to use APISIX

like what skywalking does. refer: https://github.com/apache/skywalking-website/blob/fe54f2541fcabba6ee49c6497ca9daaed2a91795/themes/docsy/layouts/partials/scripts.html#L34

image


What do you think? @moonming

Comment (zll600) [reply blog/en/docusaurus.config.js]: This repo can be divided into three parts: `website`, `doc` and `blog(en and zh)`. Each parts contains different documents, do we need different questions for different parts?

Comment (moonming) [reply blog/en/docusaurus.config.js]: > I think we can follow the user journey.
> 
> > What is APISIX -> How to deploy APISIX -> How to use APISIX
> 
> 
> 
> like what skywalking does. refer: https://github.com/apache/skywalking-website/blob/fe54f2541fcabba6ee49c6497ca9daaed2a91795/themes/docsy/layouts/partials/scripts.html#L34
> 
> 
> 
> image
> 
> 
> 
> 
> 
> What do you think? @moonming 

Good answers come from the right questions. I don't think general questions are valuable

Comment (zll600) [reply blog/en/docusaurus.config.js]: > > I think we can follow the user journey.
> > > What is APISIX -> How to deploy APISIX -> How to use APISIX
> > 
> > 
> > like what skywalking does. refer: https://github.com/apache/skywalking-website/blob/fe54f2541fcabba6ee49c6497ca9daaed2a91795/themes/docsy/layouts/partials/scripts.html#L34
> > image
> > What do you think? @moonming
> 
> Good answers come from the right questions. I don't think general questions are valuable

Very good point. 

But what I mean here is that we can set up the problem from these aspects. Didn't mean to set the specific question up as `How to use APISIX`, `How to deploy APISIX`, etc. We need to set up some valuable questions from the user's perspective to help users use APISIX more effectively, right?

The following questions you offered are all valuable questions. But there is no relationship between them.  These questions cannot provide a complete user journey. 
- What is Apache APISIX?
- What protocols can Apache APISIX handle?
- How to proxy WebSocket connection?
- How does Apache APISIX work with Keycloak?

The questions set by skywalking are related to each other, and they provide a complete user journey. This is what I want to add to APISIX.

> If there is something wrong with my thoughts, please point it out.

Comment (moonming) [reply blog/en/docusaurus.config.js]: OK, got it. You can make a try😀

**Review by moonming — APPROVED** [review]

**Review by juststillthinking — APPROVED** [review]

**Pull request merged**